README.md

```diff
@@ -32,7 +32,11 @@ To build PromLens from source, you need:
 
 * Go [version 1.17 or greater](https://golang.org/doc/install).
 * NodeJS [version 16 or greater](https://nodejs.org/).
+  * Note: With NodeJS v17+ you may experience an `error:0308010C:digital envelope routines::unsupported` error. This can be worked around by either exporting the following environment variable `NODE_OPTIONS=--openssl-legacy-provider` (please be aware of security considerations) or downgrading to NodeJS v16 [source](https://stackoverflow.com/a/69699772).
 * npm [version 7 or greater](https://www.npmjs.com/).
+* make (https://www.gnu.org/software/make/)
```

Comment:
Hmm, I'm not sure where to draw the line in terms of listing common Unix utilities as dependencies (we usually don't, for Prometheus for example: https://github.com/prometheus/prometheus#building-from-source). If we go that route, there is even more we could add, like `tar`, `gzip`, and maybe more.

Reply:
I agree, we can't list every dependency in the world. At most we should make missing tools error in an obvious way.

Reply:
That is a very fair point.

For reference, these are the errors you currently get when these dependancies are missing:

**bzip**
```
Run `npm audit` for details.
>> bundling npm licenses
rm -f "npm_licenses.tar.bz2"
find  -iname "license*" | tar cfj "npm_licenses.tar.bz2" --transform 's/^/npm_licenses\//' --files-from=-
/bin/sh: 1: bzip2: not found
tar: npm_licenses.tar.bz2: Cannot write: Broken pipe
tar: Child returned status 127
tar: Error is not recoverable: exiting now
make: *** [Makefile:41: npm_licenses] Error 2
```

**curl**
```
>> bundling npm licenses
rm -f "npm_licenses.tar.bz2"
find  -iname "license*" | tar cfj "npm_licenses.tar.bz2" --transform 's/^/npm_licenses\//' --files-from=-
curl -s -L https://github.com/prometheus/promu/releases/download/v0.13.0/promu-0.13.0.linux-amd64.tar.gz | tar -xvzf - -C /tmp/tmp.IXPE2L87O4
/bin/sh: 1: curl: not found

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now
make: *** [Makefile.common:236: /root/go/bin/promu] Error 2
```

Instead of listing those dependancies individually, we could instead add a generic statement, e.g. "This is a non-exhaustive list of dependancies, as it is assumed some standard Unix tools are available on your system. Please review build error messages for more information on missing tools."

Reply:
Yeah, that sounds good to me!

Reply:
Thanks both, I've added a new commit which executes the above changes. Thanks for the advice.

I think this conversation should be okay to resolve now (and the PR merged?).